mo_net/protos.py

```
@@ -2,7 +2,18 @@
 from collections.abc import Callable, Sequence
 from enum import StrEnum
-from typing import Generic, NewType, Protocol, Self, TypedDict, TypeVar, cast, Optional, Type as TypingType, Union
+from typing import (
+    Generic,
+    NewType,
+    Protocol,
+    Self,
+    TypedDict,
+    TypeVar,
+    cast,
+    Optional,
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import

(F401)